api/v1alpha1/osbuildenvconfig_types.go

```diff
@@ -97,11 +97,11 @@ type ExternalWorkerConfig struct {
 	// User is the user to connect with to external worker machine
 	// +kubebuilder:validation:Required
 	User string `json:"user"`
-	// RedHatCredsSecretReference is a reference to a secret in the same namespace,
+	// SSHKeySecretReference is a reference to a secret in the same namespace,
```

Comment:
Please update the sample file

Reply:
The sample file doesn't use this yet. Writing a sample for the External Worker is how I saw this error

Reply:
Also maybe need to edit the README file

Reply:
Same for the README

Reply:
https://github.com/project-flotta/osbuild-operator/blob/main/config/samples/osbuilder_v1alpha1_osbuildenvconfig.yaml#L16